csrc/predicate_compute.cpp

```diff
@@ -194,8 +194,9 @@ ParallelizedDomainPredicate::getPredicateMap(
       // If it's a root domain, it should be covered by the root
       // predicates, so no extra predicate is required.
       if (std::find(
-              tv->getRootDomain().begin(), tv->getRootDomain().end(), tv_id) !=
-          tv->getRootDomain().end()) {
+              tv->getMaybeRootDomain().begin(),
```

Comment:
Is `getMaybeRootDomain`->`getRFactorDomain` a no op? If not, we can open a separate PR for it. I'd like to keep this PR strictly no op so that the codegen diff tool can be used to sanitize this PR.

Reply:
To be clear, I agree that it should be `getRFactorDomain`, just want to be sure that it is a no op.

Reply:
fixed